needy/utility.py

```diff
@@ -1,6 +1,25 @@
+import os
+
+from contextlib import contextmanager
+
+
 class DummyContextManager():
     def __enter__(self):
         return None
 
     def __exit__(self, exc_type, exc_value, traceback):
         return False
+
+
+@contextmanager
+def activity_section(name):
```

Comment:
i have a naming request: `log_section` instead of `activity_section` and `section_name` instead of `activity_name`